packages/v3/src/components/CvNotification/CvToastNotification.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div
+    role="alert"
+    :kind="kind"
+    :class="[
+      `${carbonPrefix}--toast-notification`,
+      `${carbonPrefix}--toast-notification--${kind}`,
+      { [`${carbonPrefix}--toast-notification--low-contrast`]: lowContrast },
+    ]"
+  >
+    <component
+      :is="icon"
+      :class="`${carbonPrefix}--toast-notification__icon`"
+    />
+    <div :class="`${carbonPrefix}--toast-notification__details`">
+      <h3 :class="`${carbonPrefix}--toast-notification__title`">{{ title }}</h3>
+      <div :class="`${carbonPrefix}--toast-notification__subtitle`">
+        {{ subTitle }}
+      </div>
+      <div :class="`${carbonPrefix}--toast-notification__caption`">
+        {{ caption }}
+      </div>
+    </div>
```

Comment:
The react version seems to allow children after subtitle and caption. We should add a slot.